### PR TITLE
Update oba-efo.sssom.tsv

### PR DESCRIPTION
**Pull request by rays22**

### CODE REVIEW

File: src/mappings/oba-vt-mapping-exclusions.sssom.tsv [hunk 1]
```diff
@@ -1 +1,6 @@
-subject_id	subject_label	predicate_id	object_id	match_type	comment
\ No newline at end of file
+subject_id	subject_label	predicate_id	object_id	match_type	comment
+VT:0000023	ear orientation quality	skos:exactMatch	OBA:VT0010274	Lexical	owl:deprecated
```
Comment (matentzn) [inline src/mappings/oba-vt-mapping-exclusions.sssom.tsv:2]: does owl:deprecated mean that the VT or OBA term is deprecated?

**Review by matentzn — COMMENTED** [review]
- Did you manually remove all these rows from the unmapped/unreviewed etc tables? Or did you rerun the pipeline?
- There are many removals in this PR but very few additions! Will you make a new PR for the dosdp stuff?

File: src/mappings/oba-vt-mapping-exclusions.sssom.tsv (hunk 1 above)
Comment (rays22) [reply src/mappings/oba-vt-mapping-exclusions.sssom.tsv:2]: > does owl:deprecated mean that the VT or OBA term is deprecated?

Yes, the term in the subject_id column is deprecated. For example,
``` obo
[Term]
id: VT:0000023
name: ear orientation trait
def: "OBSOLETE. Any measurable or observable characteristic related to the location or placement of the outer ears in relation to each other or to the skull.\nreplace with VT:0010274" [MGI:cwg, MGI:MP]
is_obsolete: true
```
This is based on [the latest release of VT](https://raw.githubusercontent.com/AnimalGenome/vertebrate-trait-ontology/master/current_release/vt.obo)

Comment (matentzn) [reply src/mappings/oba-vt-mapping-exclusions.sssom.tsv:2]: perfect thank you @rays22 excellent

**Review by matentzn — APPROVED** [review]

**Pull request merged**